app/src/main/java/org/mozilla/fenix/onboarding/WallpaperOnboardingDialogFragment.kt

```diff
@@ -80,6 +80,7 @@ class WallpaperOnboardingDialogFragment : BottomSheetDialogFragment() {
         container: ViewGroup?,
         savedInstanceState: Bundle?,
     ): View = ComposeView(requireContext()).apply {
+        this@WallpaperOnboardingDialogFragment.dialog?.setCanceledOnTouchOutside(true)
```

Comment:
what if we just do `isCancelable = true` in `onCreate`?

Reply:
That's actually not working for me, oddly enough